bins/fetch/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "pevm-fetch"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+bincode.workspace = true
+clap = { workspace = true, features = ["derive"] }
+flate2.workspace = true
+serde_json = { workspace = true, features = ["preserve_order"] }
```

Comment:
Good features all crates enable by default can be better declared in the workspace `Cargo.toml`?

Reply:
You mean, that I should enable some features on workspace level, right? Should I add smth else, apart from `serde_json`?

Reply:
Yes, let's port all feature choices to the workspace `Cargo.toml` for simplicity.

Reply:
You've only done it for `pevm`, let's do the same here.